aepsych/strategy.py

```diff
@@ -332,9 +328,41 @@ def get_max(
             self.model is not None
         ), "model is None! Cannot get the max without a model!"
         self.model.to(self.model_device)
-        return self.model.get_max(
-            constraints, probability_space=probability_space, max_time=max_time
+        return self.get_max_(
+            self.model,
+            constraints,
+            probability_space=probability_space,
+            max_time=max_time,
+        )
+
+    def get_max_(
```

Comment:
The actual functions should go outside of the strategy class, these should utility functions from the model subdirectory that is imported and called by the corresponding function in the strategy class.

Reply:
I agree, these used to be methods inside `models/base.py` and I initially just moved them to `Strategy`, but I think there is no need to have them as we can directly call the functions in `models/utils.py`, which simplifies the whole structure. To do so, I had to re-implement `get_jnd() ` there.